src/software/simulation/standalone_simulator.h

```diff
@@ -47,7 +48,8 @@ class StandaloneSimulator
     /**
      * Adds robots to predefined locations on the field
      */
-    void setupInitialSimulationState();
+    void setupInitialSimulationStateDivA();
+    void setupInitialSimulationStateDivB();
```

Comment:
isn't this more about setting up 11 robots?

Reply:
Yeah that's basically everything this does